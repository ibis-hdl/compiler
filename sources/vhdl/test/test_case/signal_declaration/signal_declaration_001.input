signal my_word : word := X"01234567";
