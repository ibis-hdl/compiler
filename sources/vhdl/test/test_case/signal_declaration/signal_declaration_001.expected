(signal_declaration
  (identifier_list
    (identifier
      my_word
    )
  )
  (subtype_indication
    (type_mark<v>
      (name<v>
        (identifier
          word
        )
      )
    )
  )
  (expression
    (relation
      (shift_expression
        (simple_expression
          (term
            (primary<v>
              (literal<v>
                (bit_string_literal
                  base: HEX, literal: 01234567
                )
              )
            )
          )
        )
      )
    )
  )
)
